src/cli/cluster/cluster_manager.js

```diff
@@ -86,7 +86,6 @@ module.exports = class ClusterManager {
     const fromRoot = require('../../utils/fromRoot');
 
     this.watcher = chokidar.watch([
-      'src/plugins',
```

Comment:
I think that changing everything to absolute (including the extras that are merged) using the path module's `resolve()` function, then de-duplicating the array with `uniq` is the way to go then. This way we won't run into this issue in the future either.